src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -47,8 +47,8 @@
 // TODO: References to Lecture kind string literal need to be updated once Lecture code is merged.
 @WebServlet("/discussion")
 public class DiscussionServlet extends HttpServlet {
-  private static final String PARAM_LECTURE = "lecture";
-  private static final String PARAM_PARENT = "parent";
+  public static final String PARAM_LECTURE = "lecture";
```

Comment:
Does @visiblefortesting work here?

Reply:
IDK.  Gonna wait until we figure out if we're using a util/consts class for all of these

Reply:
So these need to be public because of tests? In that case I would make these private and just duplicate the variables in the tests. The issue with making these public is that you end up allowing other classes to use them. 

+1 to what Jay said. I would try @VisibleForTesting because duplicating the variable in tests.